packages/eslint-plugin/rules/no-top-level-material-ui-4-imports.js

```diff
@@ -16,16 +16,72 @@
 
 // @ts-check
 
+/**
+ * @typedef {object} FixerValues
+ * @property {string} value
+ * @property {string} [alias]
+ * @property {string} [componentValue]
+ * @property {string} [componentAlias]
+ * @property {boolean} emitComponent
+ * @property {boolean} emitProp
+ */
+
 const KNOWN_STYLES = [
```

Comment:
added almost all the exports from the `@material-ui/core/styles` here. not sure if its better to keep adding missing ones instead for any reason

Reply:
This is fine, this lint rule is intended to help with transition and not meant to be too permanent.